src/components/drawing/index.js

```diff
@@ -235,8 +235,10 @@ Object.keys(SYMBOLDEFS).forEach(function(k) {
     } else {
         drawing.symbolList.push(
             n + 200,
+            String(n + 200),
             k + '-dot',
             n + 300,
+            String(n + 300),
```

Comment:
OIC this is needed for coerce to work... then we also need strings up above (L222, 224) for the `n` and `n + 100` variants.

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/5073/commits/5b7b18f59ffc3c4ccb44fd24969ff2b40790c44e.